.github/workflows/deploy_stage_regions.yml

```diff
@@ -0,0 +1,107 @@
+# The Licensed Work is (c) 2022 Sygma
+# SPDX-License-Identifier: LGPL-3.0-only
+
+name: sygma_deploy_devnet_to_regions
+
+on:
+  workflow_dispatch:
```

Comment:
should not it be on push, the same it was before?

Reply:
workflow dispatch does not need push commit. If we are going to deploy manually, then the commit is not manual.